CHANGELOG.rst

```diff
@@ -2,6 +2,42 @@
 CHANGELOG
 =========
 
+2.9.0
+=====
+
+**ENHANCEMENTS**
+
+* Added support for heterogeneous instance types with Slurm Scheduler.
+* Added CLI utility to convert ParallelCluster configuration files from single to Heterogeneous instance types.
```

Comment:
Add CLI utility <command-name> to convert configuration files with Slurm scheduler to new forrmat to support multiple queue configuration.

Also move this at the bottom (or close to the bottom) of the Enh section